pkg/builtins/signature.go

```diff
@@ -0,0 +1,100 @@
+package builtins
+
+import (
+	"github.com/lambdaclass/cairo-vm.go/pkg/lambdaworks"
+	"github.com/lambdaclass/cairo-vm.go/pkg/starknet_crypto"
+	"github.com/lambdaclass/cairo-vm.go/pkg/vm/memory"
+	"github.com/pkg/errors"
+)
+
+const SIGNATURE_BUILTIN_NAME = "signature"
+
+// Notice changing this to any other number breaks the code
+const CELLS_PER_INSTANCE = 2
```

Comment:
```suggestion
const SIGNATURE_CELLS_PER_INSTANCE = 2
```
All builtins are implemented in the same package, we need to differentiate the constants to avoid name clashes

Reply:
Fixed